pyanaconda/core/path.py

```diff
@@ -147,3 +147,8 @@ def conditional_chown(path, uid, gid, from_uid=None, from_gid=None):
         dir_tree_map(root, lambda path: conditional_chown(path, uid, gid,
                                                           from_uid_only,
                                                           from_gid_only))
+
+
+def parent_dir(directory):
+    """Return the parent's path"""
+    return "/".join(os.path.normpath(directory).split("/")[:-1])
```

Comment:
Could you please try to replace the implementation with one of the solutions in https://stackoverflow.com/questions/2860153/how-do-i-get-the-parent-directory-in-python?

Reply:
This is used only in users so the implementation does not really matter as much. There are lots of rules about the parent, but it's overspecified for no reason.

Reply:
I tried, see non-review comment at the end.